common/build/eslint-config-fluid/eslint7.js

```diff
@@ -214,7 +217,6 @@ module.exports = {
         ],
 
         // eslint-plugin-unicorn
-        "unicorn/better-regex": "error",
```

Comment:
Double check that you intended to remove this config?

Reply:
Oops!